CMakeLists.txt

```diff
@@ -25,7 +25,7 @@ if(MSVC)
 	endif()
 elseif(CMAKE_COMPILER_IS_GNUCC OR CMAKE_COMPILER_IS_GNUCXX)
 	# Update if necessary
-	set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -Wall -Wno-long-long -Wno-reorder -pedantic ")
+	set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -Wall -Wno-long-long -Wno-reorder -pedantic -std=c++1y")
```

Comment:
sure

There is a new pull request

On Fri, Sep 11, 2015 at 1:27 PM, ypnos <notifications@github.com> wrote:

> In CMakeLists.txt
> <https://github.com/gerbilvis/gerbil/pull/30#discussion_r39262355>:
>
> > @@ -25,7 +25,7 @@ if(MSVC)
> >  	endif()
> >  elseif(CMAKE_COMPILER_IS_GNUCC OR CMAKE_COMPILER_IS_GNUCXX)
> >  	# Update if necessary
> > -	set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -Wall -Wno-long-long -Wno-reorder -pedantic ")
> > +	set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -Wall -Wno-long-long -Wno-reorder -pedantic -std=c++1y")
>
> Can you please change to std=c++11 as this is the standard we are working
> with.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/gerbilvis/gerbil/pull/30/files#r39262355>.
>



-- 
Ricardo Ribalda